packages/react/README.md

```diff
@@ -0,0 +1,25 @@
+# @river-build/react
```

Comment:
sorry to be picky, but can we call it `@river-build/react-sdk`?

Reply:
sure :))